armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -765,6 +766,14 @@ def test_coldAssemblyHeight(self):
                     checkColdBlockHeight(bStd, bExp, self.assertEqual, "the same")
                 else:
                     checkColdBlockHeight(bStd, bExp, self.assertNotEqual, "different")
+                if bStd.hasFlags(Flags.FUEL):
+                    # fuel mass should grow because heights are considered cold heights
+                    # and a cold 1 cm column has more mass than a hot 1 cm column
+                    if not isinstance(
+                        bStd.getComponent(Flags.FUEL).material, custom.Custom
+                    ):
+                        # custom materials don't expand
+                        self.assertGreater(bExp.getMass("U235"), bStd.getMass("U235"))
```

Comment:
Just a note to my future self. 

Since the heights in the blueprints are the same, the mass of `inputHeightsConsideredHot: False` (bExp) will be higher than `inputHeightsConsideredHot: True` (bStd). The former is cooler and therefore has a higher density than its hot counterpart - since the two cases have the same volume, the former has a higher mass. 

When bExp is expanded at BOL, mass is conserved. So post expansion, the mass is still higher than bStd.